components/builder-jobsrv/src/config.rs

```diff
@@ -34,6 +34,11 @@ pub struct Config {
     /// List of net addresses for routing servers to connect to
     pub routers: Vec<RouterAddr>,
     pub datastore: DataStoreCfg,
+
+    /// Directory to which log output of running build processes will
+    /// be written. Defaults to the system temp directory. Must exist
+    /// and be writable by the server process.
+    pub log_dir: String,
```

Comment:
I'd recommend using `PathBuf` in place of `String` when dealing with a filepath. It's worth noting that `PathBuf` == `String` and `Path` == `str`, this was something that didn't click for me for a few months in